Sources/NIOAsyncAwaitDemo/AsyncChannelIO.swift

```diff
@@ -15,7 +15,7 @@
 import NIOCore
 import NIOHTTP1
 
-#if compiler(>=5.5)
+#if compiler(>=5.5) && canImport(_Concurrency)
```

Comment:
Lgtm but we could also try `#if compiler(>=5.5) && $AsyncAwait`. That’s what the original version of this code had, no idea when we lost that. Tbh I don’t know if `$AsyncAwait` or `canImport(_Concurrency)` is more correct here. CC @dgregor

Reply:
The `#if compiler(>=5.5) && canImport(_Concurrency)` is correct, because you need both the 5.5 compiler (all released versions and betas support async/await) and the _Concurrency module.